### PR TITLE
Persist absentees as wel as participants

### PR DESCRIPTION
**Pull request by abeforgit**
fixes https://binnenland.atlassian.net/browse/GN-1810
linked:
https://github.com/lblod/app-gelinkt-notuleren/pull/19
https://github.com/lblod/notulen-prepublish-service/pull/18

### CODE REVIEW

**Comment by abeforgit** [comment]
still need to update the prepublisher

**Comment by abeforgit** [comment]
also just realised I still need to do behandelingen, so disregard for now

File: app/components/participation-list.js [hunk 1]
```diff
@@ -37,13 +51,14 @@ export default class ParticipationListComponent extends Component {
...
-      const sorted=notPresent.sortBy('isBestuurlijkeAliasVan.achternaam');
-      return sorted;
-    }
-    return [];
+    // if(this.aanwezigenBijStart.length > 0 && this.possibleParticipants) {
+    //   const aanwezigenUris = this.aanwezigenBijStart.map((mandataris) => mandataris.uri);
+    //   const notPresent = this.possibleParticipants.filter((mandataris) => !aanwezigenUris.includes(mandataris.uri));
+    //   const sorted=notPresent.sortBy('isBestuurlijkeAliasVan.achternaam');
+    //   return sorted;
+    // }
```
Comment (lagartoverde) [inline app/components/participation-list.js:59]: Why we comment out this part of the code? If it's not needed I would remove it

File: app/components/participation-list/modal.js [hunk 2]
```diff
@@ -1,20 +1,42 @@
 import Component from '@glimmer/component';
 import { action } from "@ember/object";
-import { tracked } from '@glimmer/tracking';
+import { tracked } from 'tracked-built-ins';
```
Comment (lagartoverde) [inline app/components/participation-list/modal.js:3]: What's the difference between glimmer tracking and tracked-built-ins?

**Review by lagartoverde — APPROVED** [review]
Looks good to me, just check the 2 comments I added :)

File: app/components/participation-list/modal.js (hunk 2 above)
Comment (abeforgit) [reply app/components/participation-list/modal.js:3]: https://github.com/pzuraq/tracked-built-ins
Tracked-built-ins can track things like native Maps and Arrays
The tracked decorator they provide works like the normal decorator for normal values, but can then also be used to track the builtin objects

File: app/components/participation-list.js (hunk 1 above)
Comment (abeforgit) [reply app/components/participation-list.js]: Because I forgot :stuck_out_tongue:

**Pull request merged**